src/dotnet/commands/dotnet-new/FSharp_Console/project.json.template

```diff
@@ -16,8 +16,15 @@
   },
 
   "tools": {
-       "dotnet-compile-fsc": "1.0.0-*"
-   },
+    "dotnet-compile-fsc": {
+      "version": "1.0.0-*",
+      "imports": [
```

Comment:
These are the imports which make this work. 

I'll take a look at what the failures were without them.